Rakefile

```diff
@@ -89,7 +89,7 @@ namespace :spec do
       # Set the rate limit to a large number to avoid 429: Too Many Requests errors
       # which can occur as cypress very quickly hits a lots of endpoints.
       puts "\n== Removing rate limit =="
-      exit $?.exitstatus unless system("bundle exec rails runner 'MiqServer.my_server.add_settings_for_resource(:server => {:rate_limiting => {:request => {:limit => 99999}}})'")
+      exit $?.exitstatus unless system("bundle exec rails runner 'MiqServer.my_server.add_settings_for_resource(:server => {:rate_limiting => {:request => {:limit => 99999}, :ui_login => {:limit => 99999}}})'")
```

Comment:
Alternatively, maybe we can encode this into the rate limiter itself [here](https://github.com/Fryguy/manageiq/blob/d67e6c865c1bcde98442301b1b0f7a14e119891d/config/initializers/rack_attack.rb#L9-L14) with the numbers or [here](https://github.com/Fryguy/manageiq/blob/d67e6c865c1bcde98442301b1b0f7a14e119891d/config/initializers/rack_attack.rb#L1) with the conditional in the first place?  I never liked this line because it changes settings.